datastore/models.go

```diff
@@ -571,6 +572,7 @@ type Event struct {
 	ProjectID        string                `json:"project_id,omitempty" db:"project_id"`
 	Endpoints        pq.StringArray        `json:"endpoints" db:"endpoints"`
 	Headers          httpheader.HTTPHeader `json:"headers" db:"headers"`
+	QueryParams      url.Values            `json:"query_params" bson:"query_params"`
```

Comment:
@surendratiwari3 We switched to `PostgreSQL` as our default store, you need to switch this up for it to work.